phpunit.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/8.5/phpunit.xsd"
+         colors="true"
+         bootstrap="tests/Application/config/bootstrap.php">
+    <testsuites>
+        <testsuite name="SyliusPriceHistoryPlugin Test Suite">
```

Comment:
It would be probably better to use snake case for the suite name

Reply:
I was inspired by these example settings:

https://github.com/Sylius/Sylius/blob/1.13/phpunit.xml.dist#L11
https://github.com/Sylius/Plus/blob/main/phpunit.xml.dist#L8
https://github.com/Sylius/PayPalPlugin/blob/1.5/phpunit.xml.dist